### PR TITLE
Apply actions for general type dialogs correctly

### PR DESCRIPTION
**Pull request by piroor**
Auto Confirm 0.4 cannot perform "accept" and other dialog-button-related actions. (regression on 0.4 from my previous pull request)

### CODE REVIEW

File: modules/main.js [hunk 1]
```diff
@@ -203,12 +203,24 @@ function processAction(aWindow, aAction, aRootElement)
...
   switch (action) {
   case 'accept':
-    commonDialog.ui.button0.click();
     log("accept");
+    if (commonDialog) {
+      commonDialog.ui.button0.click();
+    } else if (typeof root.acceptDialog == "function") {
+      root.acceptDialog();
+    } else {
+      Cu.reportError(new Error("We don't know how to accept "+root));
```
Comment (myokoym) [inline modules/main.js]: How about using `root.id` instead of `root`? I like element ID such as `SanitizeDialog` better than `[object XULElement]`.

**Comment by piroor** [comment]
Thank you for feedback. Now log messages has been more debug-firendly.

**Comment by myokoym** [comment]
Cool! Thank you for the quick response. I'll release the new version soon.

**Comment by piroor** [comment]
Thanks!

**Pull request merged**